shaman/util.py

```diff
@@ -155,3 +156,25 @@ def parse_distro_query(query):
             dict(distro=distro, distro_codename=codename, distro_version=version, arch=arch)
         )
     return result
+
+
+def get_repo_url(query, project_name, ref, arch, directory=None, repo_file=True):
+    # requires the repository to be fully available on a remote chacra
+    # instance for a proper redirect. Otherwise it will fail explicitly
+    if arch == 'aarch64':
```

Comment:
I took out this commit out as it was causing test failures anyway. My original goal here was to support somebody putting ``$basearch`` into a yum repo file because for an arm64 node that would print ``aarch64``.